.github/workflows/test-migrations.yml

```
@@ -0,0 +1,60 @@
+      - name: Run Local Migrations
+        run: |
+          DOTENV_FILE=.env.testing supabase db reset --linked=false  # Run migrations on local test db
+          DOTENV_FILE=.env.testing supabase db diff  # Show what changed
+
+      - name: Run Migration Tests
+        if: success()
+        run: |
+          # Add your migration tests here
+          # For example:
+          # DOTENV_FILE=.env.testing pnpm test:migrations
+          echo "Running migration tests..." 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)